metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Installs/configures Apache Cassandra'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.10.0'
+version '2.10.1'
```

Comment:
Never change version numbers in a pull request. It's up to the maintainers to decide when they should be updated and how.